nomad/plan_apply_test.go

```diff
@@ -226,6 +226,10 @@ func TestPlanApply_applyPlan(t *testing.T) {
 		t.Fatalf("missing job")
 	}
 
+	if out.ModifyTime <= 0 {
```

Comment:
Can you test that when it is created it gets a modify time too

Reply:
Done